presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -765,27 +766,27 @@ public void testArrayAgg()
     @Test
     public void testReduceAgg()
     {
-        assertQuery(
```

Comment:
I did not follow why this test needs to be changed ?

Reply:
> I did not follow why this test needs to be changed ?

Oops fixed now